js/id/core/graph.js

```diff
@@ -16,10 +16,7 @@ iD.Graph = function(other, mutable) {
 
     this.transients = {};
     this._childNodes = {};
-
-    if (!mutable) {
-        this.freeze();
-    }
+    this.frozen = !mutable;
```

Comment:
It was bothering me that a `Graph` could be initialized with `frozen` left undefined:
```
> g = iD.Graph()
> g.frozen
true
> g2 = iD.Graph(g, true)
> g2.frozen
undefined
```